packages/docs/src/locales/en/en.json

```diff
@@ -2981,86 +2980,20 @@
     }
   },
   "grid": {
-    "title": "Grid System",
```

Comment:
Maybe we can move this to separate page and add a tag in navigation menu and deprecation warning?

That's a breaking change and it's ideal if we keep backwards compatibility even within docs.